mlir/lib/Conversion/AIRToAIEPass.cpp

```diff
@@ -3200,6 +3253,8 @@ class AIRToAIEPass : public air::impl::AIRToAIEBase<AIRToAIEPass> {
       lowerAIRMemcpyOp<air::DmaMemcpyNdOp>(device, shimDmaAlloc, options);
 
       // lowerPipelineGetPut(device, tileToHerdMap);
+      if (options.trace_size > 0)
+        createTracePacketFlow(device);
```

Comment:
Can you also add this to [runTestPatterns](https://github.com/Xilinx/mlir-air/blob/main/mlir/lib/Conversion/AIRToAIEPass.cpp#L3016), along with a unit test in `mlir/test/Conversion/AIRToAIE`? The run line in the unit test might look something like this:
```mlir
// RUN: air-opt %s -air-to-aie='test-patterns=insert-trace-packet-flow trace_offset=65536 trace_size=32768'
```

Reply:
Agreed. Adding more CI tests would be very helpful, so that for example future code development wouldn't silently break this feature.